tt_eager/tt_lib/csrc/tt_lib_bindings_tensor_dm_ops.cpp

```diff
@@ -178,7 +178,7 @@ namespace tt::tt_metal::detail{
         m_tensor.def("untilize", &untilize,
             py::arg("input").noconvert(),
             py::arg("output_mem_config").noconvert() = operation::DEFAULT_OUTPUT_MEMORY_CONFIG,
-            py::arg("use_multicore").noconvert() = false,
+            py::arg("use_multicore").noconvert() = true,
```

Comment:
I think this was changed to `false` default by @tt-aho / @TT-BrianLiu . Need to ask them if there was a particular reason for that and if we can change it back to `true`

Reply:
This might be the culprit for the CI failures... Testing now